src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const normalizedQuery = query.toLowerCase().trim();
+  const isMatch = (text: string): boolean => {
+    return text.toLowerCase().includes(normalizedQuery);
+  };
+
+  const visibleMovies = moviesFromServer.filter(
+    movie => isMatch(movie.title) || isMatch(movie.description),
+  );
```

Comment:
don't forget about memoization, so  you need to use hook useMemo() here

Reply:
Maybe this is one of the next topics of study? I have not yet studied this kind of hook